src/renderer/modals/FullNode/FullNodeBody.js

```diff
@@ -0,0 +1,167 @@
+import type { RPCNodeConfig } from "@ledgerhq/live-common/lib/families/bitcoin/satstack";
+import { ModalBody } from "~/renderer/components/Modal";
+import TrackPage from "~/renderer/analytics/TrackPage";
+import Breadcrumb from "~/renderer/components/Stepper/Breadcrumb";
+import StepLanding, { StepLandingFooter } from "./steps/StepLanding";
+import StepNode, { StepNodeFooter } from "./steps/StepNode";
+import StepAccounts, { StepAccountsFooter } from "./steps/StepAccounts";
+import StepConnectDevice, { StepDeviceFooter } from "./steps/StepConnectDevice";
+import StepSatStack, { StepSatStackFooter } from "./steps/StepSatStack";
+import StepDisconnect, { StepDisconnectFooter } from "./steps/StepDisconnect";
+import type { FullNodeSteps, ConnectionStatus } from "~/renderer/modals/FullNode";
+import { connectionStatus } from "~/renderer/modals/FullNode";
+import { saveLSS } from "./helpers";
+import { getEnv } from "@ledgerhq/live-common/lib/env";
+import { loadLSS } from "~/renderer/modals/FullNode/helpers";
+
+const steps = ["landing", "node", "accounts", "device", "satstack", "disconnect"];
+const FullNodeBody = ({
+  onClose,
+  onStepChange,
+  activeStep,
+}: {
+  onClose: any,
+  onStepChange: FullNodeSteps => void,
+  activeStep: FullNodeSteps,
+}) => {
+  const [errors, setErrors] = useState(null);
+  const satStackAlreadyConfigured = getEnv("SATSTACK");
+  const [satStackDownloaded, setSatStackDownloaded] = useState(false);
+
+  const [nodeConnectionStatus, setNodeConnectionStatus] = useState<ConnectionStatus>(
+    connectionStatus.IDLE,
+  );
+  const items = [
+    { label: <Trans i18nKey={"fullNode.modal.steps.node.title"} /> },
+    { label: <Trans i18nKey={"fullNode.modal.steps.accounts.title"} /> },
+    { label: <Trans i18nKey={"fullNode.modal.steps.device.title"} /> },
+    { label: <Trans i18nKey={"fullNode.modal.steps.satstack.title"} /> },
+  ];
+
+  const [numberOfAccountsToScan, setNumberOfAccountsToScan] = useState(10);
+  const [nodeConfig, setNodeConfig] = useState<RPCNodeConfig>({
+    host: "",
+    username: "",
+    password: "",
+    notls: false,
+  });
+  const [scannedDescriptors, setScannedDescriptors] = useState();
+  const patchNodeConfig = useCallback((patch: any) => setNodeConfig({ ...nodeConfig, ...patch }), [
+    nodeConfig,
+  ]);
+
+  const errorSteps = errors ? [1] : [];
+  const showBreadcrumb = !["landing", "disconnect"].includes(activeStep);
+  const stepIndex = steps.indexOf(activeStep) - 1;
+
+  const onAcceptLanding = useCallback(() => onStepChange("node"), [onStepChange]);
+
+  const validNodeConfig = useMemo(() => {
+    const errors = validateRPCNodeConfig(nodeConfig);
+    // NB All fields are required, why do we need these errors?
+    return !errors.length;
+  }, [nodeConfig]);
+
+  useEffect(() => {
+    // Prefill the node data for edit?
+    async function prefill() {
+      if (satStackAlreadyConfigured) {
+        const maybeNodeConfig = await loadLSS();
```

Comment:
maybeLSSConfig, it's not just node config in this object.

Reply:
Fair enough